src/controllers/dashboard/plugins/available/index.html

```diff
@@ -1,6 +1,15 @@
 <div id="pluginCatalogPage" data-role="page" class="page type-interior pluginConfigurationPage fullWidthContent">
     <div>
         <div class="content-primary">
+            <div class="sectionTitleContainer flex align-items-center">
+                <h2 class="sectionTitle">${TabCatalog}</h2>
+                <a is="emby-linkbutton" class="fab" href="#/dashboard/plugins/repositories" style="margin-left:1em;" title="${Settings}">
+                    <span class="material-icons settings" aria-hidden="true"></span>
+                </a>
+                <a is="emby-linkbutton" rel="noopener noreferrer" class="raised button-alt headerHelpButton" target="_blank" href="https://jellyfin.org/docs/general/server/plugins/">
```

Comment:
Just thought of this, not specific to this PR; it is much easier to maintain the various links to our documentation if we added all of them to a single file. That way they are named and easy to update if we refactor our docs.